frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Stage 1: Build the React application
-FROM node:14 AS build
+FROM node:16 AS build
```

Comment:
Is the docker working now on the frontend? And why not the latest node like 20, 21? Was there any dependency issues?

Reply:
yes it is working now. 
i will update the node version in the next pull request